README.md

```diff
@@ -1 +1,89 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
```

Comment:
- Good job using the recommended Microverse readme template, but in order to make your readme more professional, please make sure that all required section headers and their corresponding sections are present in your readme. Please check the [microverse readme](https://github.com/microverseinc/readme-template) template for all the required headers. Please fix this 🙏 
- Since you are not required to deploy this project online for this milestone, you can remove the Live Demo header and its corresponding section in the readme. Please note this 😄